### ISSUE TITLE
Feature Request: Notification Sound Options (mute, change, etc).

### ISSUE THREAD

**Issue by ahpatel** [yakyak/yakyak#405]
Can you add a way to mute and/or change the notification sound within the application?

In Hangouts, there's an option to mute notifications for 1 hr, 8 hrs, 24 hrs... Something similar would be very helpful for me because I often use my PC for VoIP calls and have to close YakYak to prevent the notification sounds interrupting the call every minute.

**Comment by averissimo** [comment]
Great idea, muting should be a matter of creating an option and checking in `notifications.coffee` around line 65 (not sure if it's there though)

```
...
sender: 'com.github.yakyak'
sound: viewstate.someOptionThatMutesSounds == false
, (err, res) ->
...
```

To mute it for some hours, would require to store in that option the date/time of the "unmute"

```
...
sender: 'com.github.yakyak'
sound: (!viewstate.someOptionThatMutesSounds || viewstate.someOptionThatMutesSounds < getTime())
, (err, res) ->
...
```

**Comment by averissimo** [comment]
@ahpatel I'm actually working on a feature to add sound to notifications engines that don't support it and an option to disable sound all together

since you created the issue a new option was developed to turn notifications on or off in the tray icon and `view` menu

try the beta release of 1.4.0 at #451

**Comment by averissimo** [comment]
partial duplicate of #381

**Comment by averissimo** [comment]
@ahpatel mute is implemented in pull request #467
